src/ipyaladin/__init__.py

```diff
@@ -29,7 +29,8 @@ class Aladin(anywidget.AnyWidget):
 
     # Options for the view initialization
     height = Int(400).tag(sync=True, init_option=True)
-    target = Unicode("0 0").tag(sync=True, init_option=True)
+    _target = Unicode("0 0").tag(sync=True, init_option=True)
+    _frame = Unicode("icrs").tag(sync=True, init_option=True)
```

Comment:
There is already a coo_frame trait that you might be able to use here

Reply:
The frame traitlets was used in previous testing, I will remove it